pkg/controlplane/apiserver/server.go

```
@@ -146,6 +147,28 @@ func (c completedConfig) New(name string, delegationTarget genericapiserver.Dele
 		return nil, fmt.Errorf("failed to get listener address: %w", err)
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(apiserverfeatures.CoordinatedLeaderElection) {
+		_ = s.VersionedInformers.Coordination().V1().Leases().Informer()
+		_ = s.VersionedInformers.Coordination().V1alpha1().LeaseCandidates().Informer()
+		// What's the best way of triggering these informer creations.
+		// We can only figure out if we won the lease after poststart.
+
+		s.GenericAPIServer.AddPostStartHookOrDie("start-kube-apiserver-coordinated-leader-election-controller", func(hookContext genericapiserver.PostStartHookContext) error {
+			_, err := leaderelection.NewGenerator(
+				hookContext,
+				s.GenericAPIServer.LoopbackClientConfig,
+				s.VersionedInformers.Coordination().V1().Leases(),
+				s.VersionedInformers.Coordination().V1alpha1().LeaseCandidates(),
+				client.CoordinationV1(),
+				client.CoordinationV1alpha1(),
+			)
+			if err != nil {
+				runtime.HandleError(err)
```

doesn't this hide the error?